src/locale/locales/ca/messages.po

```diff
@@ -298,7 +298,7 @@ msgstr "Afegeix text alternatiu"
 
 #: src/view/com/composer/GifAltText.tsx:175
 msgid "Add ALT text"
-msgstr ""
+msgstr "Afegeix ALT"
```

Comment:
The English string was a typo that was fixed in #3934 to become `Add alt text` (the message catalogs haven't been updated with it yet). So I suggest you might want to change this to follow the translation below, something like `Afegeix text alternatiu`?